### DISCUSSION TITLE
Why is there no `crypt` and `encrypt`?

### DISCUSSION THREAD

**Discussion by jonasfroeller** [neondatabase/neon#8373] [Q&A] [answered]
It works in the webui, but doesn't work, when I connect with drizzle, hosted on vercel (postgres.js).

**Comment by danieltprice** [comment]
Hi @jonasfroeller 
Have you installed the pgcrypto extension?
CREATE EXTENSION IF NOT EXISTS pgcrypto;

https://www.postgresql.org/docs/current/pgcrypto.html

**Reply by jonasfroeller** [reply, marked as answer]
Yes, I did that. the weird thing is, that I can only use it in the webui, and not when I connect to the database using the connection string.

I get:
```
function decrypt(bytea, unknown, unknown) does not exist
```

I used the wrong postgres driver... Just didn't think this through enough, because it seemed to work, and I wanted to be able to easily switch from my dev db to the neondb. (deployed to vercel (serverless)).  
I now use the neon serverless driver.

Nevermind. Tested it with the postgres adapter again, before pushing, which works too now, for some reason.

**Reply by jonasfroeller** [reply]
Serverless driver doesn't work. The other one does.
